docker-compose.yaml

```diff
@@ -0,0 +1,43 @@
+version: '3.3'
+services:
+  web:
+    image: "node:20"
+    user: "node"
+    working_dir: /home/node/app
+    environment:
+      - NODE_ENV=production
+    volumes:
+      - ./:/home/node/app
+    ports:
+      - "8080:8080"
+      - "8000:8000"
+      - "8081:8081"
```

Comment:
Hi, can you use the port numbers from the environment file (it will use `.env` by default).

Reply:
Nice one! Will do now.